compiler/msgs.nim

```diff
@@ -486,6 +486,28 @@ proc formatMsg*(conf: ConfigRef; info: TLineInfo, msg: TMsgKind, arg: string): s
               else: ErrorTitle
   conf.toFileLineCol(info) & " " & title & getMessageStr(msg, arg)
 
+proc colorError(s: string, color: ForegroundColor): string =
+  when not defined(colorfulError):
```

Comment:
`nimColorErrors` (see contributing.rst for rationale of nim prefix)

Reply:
Will do it's just for testing mostly at the moment.